src/_includes/layouts/event.html

```diff
@@ -1,22 +1,26 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-	<meta charset="UTF-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>{{ title }} | Lunch.dev Community Calendar</title>
-</head>
-<body>
-	{% include partials/header.html %}
-	<main>
-		<h1>{{ title }}</h1>
-		{% if speakers %}
-		<ul>
-			{% for speaker in speakers %}
-			<li>{{ speaker }}</li>
-			{% endfor %}
-		</ul>
-		{% endif %}
-		{{ content }}
-	</main>
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/reset.css" />
+    <link rel="stylesheet" href="/css/styles.css" />
+    <title>{{ title }}{% if type %}, {{type}}{% endif %} | Lunch.dev Community Calendar</title>
+  </head>
+  <body data-typeface="system-ui">
+    <div data-layout="centered-single-column">
+      {% include partials/header.html %}
+      <main>
+        <h1>{{ title }}</h1>
+        {% if speakers %}
+        <ul>
+          {% for speaker in speakers %}
+          <li>{{ speaker }}</li>
+          {% endfor %}
+        </ul>
+        {% endif %} {{ content }}
+        <p>Join this event <a href="https://discord.com/invite/q9ZdBgP">in Discord</a></p>
```

Comment:
I think this is best left for our lunch session on filters, but I'd love to show this "Join this event" link only for upcoming events, and not for past events.